common/core/desktop/include/keyman/keyboardprocessor_debug.h

```diff
@@ -61,10 +77,8 @@ typedef struct {
 typedef struct {
   uint32_t  type; // 32 bits is better optimized than 8 bits
   uint32_t flags;
-  union {
-    km_kbp_state_debug_key_info key_info;
-    km_kbp_state_debug_kmx_info kmx_info;
-  };
+  km_kbp_state_debug_key_info key_info;
+  km_kbp_state_debug_kmx_info kmx_info;
 } km_kbp_state_debug_item;
```

Comment:
If my interpretation of the `kmx_debugger.cpp` & `.h` changes is right, this structure corresponds to one entry on the debugger's equivalent of a call stack?  If so... I think everything makes sense & that those two comments I referred to earlier are not in conflict.

Reply:
Yes.